src/Microsoft.DotNet.Compiler.Common/ProjectContextExtensions.cs

```diff
@@ -37,6 +37,12 @@ public static string GetSDKVersionFile(this ProjectContext context, string confi
             return Path.Combine(intermediatePath, ".SDKVersion");
         }
 
+        public static string IncrementalCacheFile(this ProjectContext context, string configuration, string buildBasePath, string outputPath)
+        {
+            var intermediatePath = context.GetOutputPaths(configuration, buildBasePath, outputPath).IntermediateOutputDirectoryPath;
+            return Path.Combine(intermediatePath, ".IncrementalCache");
```

Comment:
I looked into it and it's not worth it. Ideally, there should be an IntermediaryOutputFiles from the OutputCalculator, and a pluginable mechanism where unknown verbs plugin new files (to know about compile-csc.rsp, compile-fsc.rsp. etc). Too much of a change for this PR.